Files/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -150,6 +150,16 @@
             <Style TargetType="uc:DataGridHeader">
                 <Setter Property="HorizontalContentAlignment" Value="Stretch" />
             </Style>
+
+            <Style
+                x:Key="ColumnContentTextBlock"
+                BasedOn="{StaticResource CaptionTextBlockStyle}"
+                TargetType="TextBlock">
+                <Setter Property="TextWrapping" Value="NoWrap" />
+                <Setter Property="TextTrimming" Value="CharacterEllipsis" />
+                <Setter Property="VerticalAlignment" Value="Center" />
+                <Setter Property="HorizontalAlignment" Value="Stretch" />
```

Comment:
What about keeping the `FontWeight="Light"` or add `Opacity="0.6"` like in TilesView?

Reply:
Good idea

Reply:
![image](https://user-images.githubusercontent.com/59544401/129261710-701cb848-c736-40bf-a712-b0b13462d58b.png)